RNIap.podspec

```diff
@@ -12,7 +12,7 @@ Pod::Spec.new do |s|
   s.license      = package["license"]
   s.authors      = package["author"]
 
-  s.platforms    = { :ios => "10.0" }
+  s.platforms    = { :ios => "10.0" , :tvos => "10.0"}
```

Comment:
It has been moved out from the react-native `core` but I would say it doesn't mean it can't work for `react-native-iap`. If one decides to use [`react-native-tvos`](https://github.com/react-native-tvos/react-native-tvos) and install `react-native-iap` I believe this line would be require to make it work? I never use react native tvos, I'm just guessing here